### ISSUE TITLE
spawn fnm ENOENT (downloaded but not in $PATH)

### ISSUE THREAD

**Issue by andrewbranch** [jakebailey/every-ts#21]
I saw the tool download fnm, but it seems to put it somewhere not in my PATH and then tries to spawn it without qualifying its location:

```
❯ every-ts bisect start
Cloning TypeScript; this may take a bit...
Cloning into '/Users/andrew/n/lib/node_modules/every-ts/.data/TypeScript'...
remote: Enumerating objects: 275701, done.
remote: Counting objects: 100% (965/965), done.
remote: Compressing objects: 100% (591/591), done.
remote: Total 275701 (delta 490), reused 692 (delta 324), pack-reused 274736
Receiving objects: 100% (275701/275701), 713.89 MiB | 13.26 MiB/s, done.
Resolving deltas: 100% (164838/164838), done.
remote: Enumerating objects: 65158, done.
remote: Counting objects: 100% (40643/40643), done.
remote: Compressing objects: 100% (30748/30748), done.
remote: Total 65158 (delta 10641), reused 10135 (delta 9895), pack-reused 24515
Receiving objects: 100% (65158/65158), 27.77 MiB | 10.18 MiB/s, done.
Resolving deltas: 100% (15562/15562), done.
Updating files: 100% (70160/70160), done.
status: waiting for both good and bad commits
Building TypeScript...
Downloading fnm...
TypeScript built successfully!

~/Developer/microsoft/eg/core (28a6f05b) !2
❯ every-ts bisect good v5.2.2
status: waiting for bad commit, 1 good commit known

~/Developer/microsoft/eg/core (28a6f05b) !2
❯ every-ts bisect bad v5.3.0-dev.20231022
Resolved v5.3.0-dev.20231022 to 8af8f3c4d1e5cc6654916de9f42e9ba370513157
Bisecting: a merge base must be tested
remote: Enumerating objects: 3299, done.
remote: Counting objects: 100% (2418/2418), done.
remote: Compressing objects: 100% (1125/1125), done.
remote: Total 3299 (delta 1528), reused 1293 (delta 1293), pack-reused 881
Receiving objects: 100% (3299/3299), 6.01 MiB | 6.64 MiB/s, done.
Resolving deltas: 100% (1918/1918), done.
[e936eb13d2900f21d79553c32a704307c7ad03dd] Update package-lock.json
Building TypeScript...
Error: Command failed with ENOENT: fnm install 20
spawn fnm ENOENT
    at ChildProcess._handle.onexit (node:internal/child_process:286:19)
    at onErrorNT (node:internal/child_process:484:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'spawn fnm',
  path: 'fnm',
  spawnargs: [ 'install', '20' ],
  originalMessage: 'spawn fnm ENOENT',
  shortMessage: 'Command failed with ENOENT: fnm install 20\nspawn fnm ENOENT',
  command: 'fnm install 20',
  escapedCommand: 'fnm install 20',
  exitCode: undefined,
  signal: undefined,
  signalDescription: undefined,
  stdout: '',
  stderr: '',
  cwd: '/Users/andrew/Developer/microsoft/eg/core',
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}
Unable to build typescript at rev e936eb13d2900f21d79553c32a704307c7ad03dd; please file a bug!

~/Developer/microsoft/eg/core (28a6f05b) !2
❯ which fnm
fnm not found
```

**Comment by jakebailey** [comment]
Hm, I definitely update `PATH` before using `fnm`... https://github.com/jakebailey/every-ts/blob/d5b31ebd6439e08df5f8e3991bc9cc93edda4397/src/fnm.ts#L64

**Comment by jakebailey** [comment]
Ah, probably some path is missing a call that ensures it's been installed; will look quick.

**Comment by jakebailey** [comment]
Ugh, no, that doesn't seem to be it. Let me just try some less complicated method to do this, e.g. execute it directly.

**Comment by jakebailey** [comment]
Can you try 1.2.0?

**Comment by andrewbranch** [comment]
Also FWIW

```
~/Developer/microsoft/eg/core (28a6f05b) !2
❯ every-ts --version
Unknown Syntax Error: Unsupported option name ("--version").
```

**Comment by andrewbranch** [comment]
1.2.0 works 👍

**Comment by jakebailey** [comment]
Uh, hm. That has to be some sort of clipanion bug; it's documented to work but apparently doesn't. Will look. But you can get the version in `every-ts -h` too.

**Comment by jakebailey** [comment]
Fixed `--version` in 1.2.1.
